src/components/Autocomplete/Autocomplete.jsx

```diff
@@ -13,6 +13,26 @@ import 'react-virtualized-select/styles.css';
 const DEFAULT_CLASS = 'ui-128__autocomplete';
 const ERROR_CLASS = 'ui-128__autocomplete--error';
 
+const Tethered = ({ children }) => {
```

Comment:
I believe making all the components in this file `function`s rather than `const`s will potentially make future debugging easier